### PR TITLE
Changelog Updates

### PR DESCRIPTION
**Pull request by rcoringrato-sfdc**
### What does this PR do?

- Changelog updates for v48.24.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+
+#### salesforcedx-vscode-core
+
+- Cancelling a command execution should not throw errors ([PR #2316](https://github.com/forcedotcom/salesforcedx-vscode/pull/2316))
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Add unlocked package support to Org Browser ([PR #2305](https://github.com/forcedotcom/salesforcedx-vscode/pull/2305))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:13]:
```suggestion
- Add unlocked package support to Org Browser ([PR #2305](https://github.com/forcedotcom/salesforcedx-vscode/pull/2305), [Issue #2204](https://github.com/forcedotcom/salesforcedx-vscode/issues/2204))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+
+- Enhance performance for extension activation ([PR #2324](https://github.com/forcedotcom/salesforcedx-vscode/pull/2324))
+
+- Add telemetry for beta deploy/retrieve operations ([PR #2299](https://github.com/forcedotcom/salesforcedx-vscode/pull/2299))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:17]: We can remove this once since it was part of last week's release.

**Comment by rcoringrato-sfdc** [comment]
@hbc-bindu @ekapner Can you review the changelog updates please?

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
+# 48.24.0 - July 8, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Cancelling a command execution should not throw errors ([PR #2316](https://github.com/forcedotcom/salesforcedx-vscode/pull/2316))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: @rcoringrato-sfdc , we had a similar entry in last week's log. Specify the error type to make it clearer?
Fix SFDX commands causing _toString_ errors when cancelled

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Add unlocked package support to Org Browser ([PR #2305](https://github.com/forcedotcom/salesforcedx-vscode/pull/2305), [Issue #2204](https://github.com/forcedotcom/salesforcedx-vscode/issues/2204))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md:13]: Support for unlocked packages in Org Browser

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,21 @@
...
+
+- Enhance performance for extension activation ([PR #2324](https://github.com/forcedotcom/salesforcedx-vscode/pull/2324))
+
+- Add namespace support for beta deploy/retrieve operations ([PR #2330](https://github.com/forcedotcom/salesforcedx-vscode/pull/2330), [Issue #2188](https://github.com/forcedotcom/salesforcedx-vscode/issues/2188), [Issue #2281](https://github.com/forcedotcom/salesforcedx-vscode/issues/2281))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md:17]: Support namespaces for beta deploy and retrieve operations

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,21 @@
...
+
+- Cancelling a command execution should not throw errors ([PR #2316](https://github.com/forcedotcom/salesforcedx-vscode/pull/2316))
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Add unlocked package support to Org Browser ([PR #2305](https://github.com/forcedotcom/salesforcedx-vscode/pull/2305), [Issue #2204](https://github.com/forcedotcom/salesforcedx-vscode/issues/2204))
+
+- Enhance performance for extension activation ([PR #2324](https://github.com/forcedotcom/salesforcedx-vscode/pull/2324))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md:15]: @rcoringrato-sfdc , would you want to add details about which extension...
Enhance performance by removing Salesforce CLI extension checks during activation. Command-level errors handle CLI installation or configuration issues, if any.

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 3 above)
Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md:7]: Yes, this is a follow up fix to last week's one.

**Pull request merged**